### DISCUSSION TITLE
Why does WebView2 add additional members to the window object?

### DISCUSSION THREAD

**Discussion by Panache1** [MicrosoftEdge/WebView2Feedback#4369] [Q&A]
As described by Manuel Caballero in [https://stackoverflow.com/questions/72147601/detecting-webview2-vs-standalone-edge-browser], the additional members added to the window by WebView2 make it easy to tell whether a user is using Edge or WebView2.  

Since so many things about WebView2 can be customized to fit the needs of the user, I'm curious as to why these are added and whether there is any way to control whether or not they are added.

**Comment by victorhuangwq** [comment]
@lflores-ms I remember you were working on the User Agent work. Do you have any insights into this question?
